### PR TITLE
sound: create sandbox example for text to speech library pull request

### PR DESCRIPTION
**Pull request by JaStarbuck**
Summary: We have successfully cloned the espeak repo into the VM and followed the demo to create the test-espeak.c file in the sandbox folder of the sound directory. After getting driver warnings from computers that don't have speakers, we were able to get it to run on a pc that has speakers completely without errors. In the future, we will hopefully be able to generalize the process in the demo to create functions to use text to speech during a chiventure game.

first comment: Added the test-espeak.c file from https://github.com/espeak-ng/espeak-ng/blob/master/docs/integration.md and cloned the espeak repo so that running ./test-espeak will output sound from the VBox. There is an accidental minor change to sound.c that can be ignored.

### CODE REVIEW

**Review by jiaoboxuan — APPROVED** [review]
Looks good. That said, the comment looks like it belongs to a `README.md` file perfectly, so I suggest you do so. Especially considering it's a sandbox, future groups could pick them up easily if they can locate a `README.md`.

File: src/sound/sandbox/test-espeak.c [hunk 1]
```diff
@@ -0,0 +1,25 @@
+#include <espeak-ng/speak_lib.h>
+ 
+// in order to run this, in the chiventure/src/sound/sandbox directory run
```
Comment (ecoble) [inline src/sound/sandbox/test-espeak.c]: Please make this a block comment like this 
/*
 *
 */

File: src/sound/sandbox/test-espeak.c [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+ 
+// in order to run this, in the chiventure/src/sound/sandbox directory run 
+// gcc test-espeak.c -lespeak-ng -o test-espeak
+// especially anytime after a change is made and then use
+// ./test-espeak
+// in order to hear the text
+
+ espeak_AUDIO_OUTPUT output = AUDIO_OUTPUT_SYNCH_PLAYBACK;
+ char *path = NULL;
+ void* user_data;
```
Comment (ecoble) [inline src/sound/sandbox/test-espeak.c]: Please switch asterisk to the other side for consistent style, you can fix style automatically using astyle, more documentation [here](https://uchicago-cs.github.io/dev-guide/style_guide_c.html#style-guide-c).

File: src/sound/sandbox/test-espeak.c [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+ unsigned int *identifier;
+ 
+ int main(int argc, char* argv[]) {
```
Comment (ecoble) [inline src/sound/sandbox/test-espeak.c]: Asterisk style

**Review by ecoble — CHANGES_REQUESTED** [review]
This looks good besides some style issues! You can leave the comment in this file for now, but a README containing this as well as installation instructions is a good idea.

**Review by jiaoboxuan — APPROVED** [review]
Looks about okay.

**Review by ecoble — APPROVED** [review]
Nice work! Excited to see the demo!

**Comment by jwschroeder21** [comment]
closing statement: 

We first successfully cloned the espeak repo into the VM.  We then followed the demo to create the test-espeak.c file in the sandbox folder of the sound directory. Then, after getting driver warnings from computers that don't have speakers, we ran it on a PC that had speakers (no errors!). 

Our goal is to generalize the process in the demo to create functions to use text to speech during a chiventure game.

With Elizabeth and Victor's approval, this PR has been successfully merged!

**Comment by ecoble** [comment]
Issue Score: Excellent

Comments:
Nice job with e-speak, integrating a new library can be really challenging and you did a great job!

**Pull request merged**